crates/pera-core/src/dwallet_mpc/mpc_instance.rs

```diff
@@ -112,6 +112,33 @@ impl DWalletMPCInstance {
         Ok(())
     }
 
+    fn restart(&mut self) {
```

Comment:
missing docstring :smile:

Reply:
private functions doesn't require docstring & i think the name is quite descriptive